docs/access.md

```diff
@@ -1,20 +1,118 @@
 # Access management
 
-## LaminDB
+This document outlines the access control system, detailing its architecture, key features, roles, and implementation guidelines.
 
-LaminDB doesn't come with access management but relies on the underlying data infrastructure for this.
+## System architecture
 
-## LaminHub
+Our access management is built on a four-tier hierarchy, providing granular control over resources:
 
-```{include} includes/access-laminhub.md
```

Comment:
Where did this document go? Please either delete if no longer useful or bring it back in some form.

Reply:
I moved it at the end and updated the content, let me know what do you think.

Reply:
I think we now have a dangling file on main: https://github.com/laminlabs/lamin-docs/blob/82a725be9e8f1844bcee976f3a1aec18bd054169/docs/includes/access-laminhub.md

> Please either delete if no longer useful or bring it back in some form.

I guess you thought it's still referenced elsewhere, but I did a search and it turns out that the one sentence at the top of the page is just hard-coded rather than imported.

<img width="807" alt="image" src="https://github.com/user-attachments/assets/9b65b6de-24fa-4fc5-b187-80d9140d0003" />

So, it's dangling. I'll remove it from main but would appreciate what you mean with this:

> I moved it at the end and updated the content